README.md

```diff
@@ -225,4 +225,16 @@ Python projects, questions are welcome on the scipy-user mailing
 list. Specialized discussions or design issues should take place on
 the PyData mailing list / Google group:
 
-https://groups.google.com/forum/#!forum/pydata
+https://mail.python.org/mailman/listinfo/pandas-dev
+
+## Contributing to pandas
+
+**[How to contribute.](http://pandas.pydata.org/pandas-docs/stable/contributing.html)**
```

Comment:
Small comment: I would put this after the "all contributions ... welcome" line, and make it a full sentence. Something like "The best starting point is the contributing guide". or "A detailed overview on how to contribute can be found in the contributing guide"